client/scripts/controllers/server/tags.ts

```diff
@@ -103,14 +104,13 @@ class TagsController {
     }
   }
 
-  public async refreshAll(communityId?, chainId?, reset = false) {
+  public async refreshAll(chainId, communityId, reset = false) {
     try {
       const response = await $.get(`${app.serverUrl()}/bulkTags`, {
-        chain: chainId,
-        community: communityId,
+        chain: chainId || app.activeChainId(),
+        community: communityId || app.activeCommunityId(),
```

Comment:
if above, these would be:
`chain: chainId || null,`
`community: communityId || null,`